addons/mail/static/src/core/web/chatter.js

```diff
@@ -379,7 +391,7 @@ export class Chatter extends Component {
         this.closeSearch();
         const schedule = async (threadId) => {
             await this.activityService.schedule(this.props.threadModel, [threadId]);
-            this.load(this.props.threadId, ["activities", "messages"]);
+            this.load(["activities", "messages"]);
```

Comment:
This kind of async call to `load` still need to specify the `thread` because otherwise it could be meant for the previous thread and reload the current one.

And I would suggest that when load receives as param a thread that is different than the current one, it does nothing.

Reply:
Actually, the previous code was doing the thing you mentioned. because it was using the `this.props.threadId`. I will change the `this.props.threadId` and load it with the `threadId`